guides/v2.1/cloud/trouble/robots-sitemap.md

```diff
@@ -1,62 +1,55 @@
 ---
 group: cloud
-subgroup: 170_trouble
 title: Add site map and search engine robots
-menu_title: Add sitemap and robots.txt
-menu_order: 40
-menu_node:
 version: 2.1
 github_link: cloud/trouble/robots-sitemap.md
 functional_areas:
   - Cloud
   - Configuration
 ---
-The `pub` directory is _read only_ in the {{site.data.var.ece}} Integration, Staging, and Production environments. An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
+An attempt to generate and write the `sitemap.xml` file to the root directory results in the following error:
 
 ```
-The path ".:///app/app/" is not writable.
-Path "/app/pub/static/sitemap.xml" is protected and cannot be used.
+Please make sure that "/" is writable by the web-server.
 ```
 
-To avoid this error, you can only write to specific directories, such as `var`, `pub/media`, or `app/etc`. When using {{site.data.var.ece}}, you must use a redirect in the _read-only_ environments.  Generate the files on your local workstation and upload them to one of the write-enabled directories using the following, appropriate environment:
+With {{site.data.var.ece}}, you can only write to specific directories, such as `var`, `pub/media`, `pub/static`,or `app/etc`. When you generate the `sitemap.xml` file using the Admin panel, you must specify the `/media/` path.
 
--  For Starter, access the `staging` branch
--  For Pro, access the `integration` branch
+For version 2.1.11 and later, you do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. You can view the content in your browser with the url: `<domain.your.project>/robots.txt`
```

Comment:
For all versions, the robots.txt content is stored in the database. But since version 2.1.11 the file is no longer created

For version 2.1.11 and later, when you create `robots.txt` using Magento Admin panel the file robots.txt will not created. Because response on the fly. You can view the content in your browser with the url: `<domain.your.project>/robots.txt`.